demo-scenarios/bandwidth-example.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -xe
+
+if [ "$#" -ne 1 ]; then
+    echo "$0 <name of the network configuration to use>"
+    exit 1
+fi
+
+bandwidth_net_name="$1"
+
+export NETCONFPATH="${NETCONFPATH:-(pwd)/examples}"
+export CNI_PATH="${CNI_PATH:-../cni-plugins/bin}"
```

Comment:
Went to run it and this path doesn't exist. Should there be an install section about building the CNI plugins? in my testing/review I'm going to change it to path where I have all the CNI plugins built.

Reply:
(or you could include the bins in the repo, I guess, for demo purposes)

Reply:
I'd put the bins in the repo directly, either that or just the default one /opt/cni/bin?